nengo_spa/algebras/tests/test_algebras.py

```diff
@@ -83,6 +84,23 @@ def test_integer_binding_power(algebra, d, rng):
     assert np.allclose(algebra.binding_power(v, 3), algebra.bind(algebra.bind(v, v), v))
 
 
+@pytest.mark.parametrize("d", [25, 36])
+def test_integer_binding_base_implementation(algebra, d, rng):
```

Comment:
Heh yeah I was going to leave a comment saying that this is only testing that they match, and then the previous unit test is checking that it is correct -- but then I noticed that none of the unit tests had comments so I felt awkward breaking that pattern.

Reply:
Ah ok, I didn't see the other test because it was cut off in the diff. I added a commit to slightly rename the test to express that it is only intended to test consistency between the concrete algebra and the base implementation. Together with the other test this should be sufficient to test the base implementation. Thus, I wouldn't go through the lengths to test the base implementation in isolation by either using a mock (which would likely coupled to the implementation) or dummy algebra (too much effort for a single test case).